opal/mca/smsc/base/base.h

```diff
@@ -14,7 +15,6 @@
 #include "opal/mca/smsc/smsc.h"
 
 extern mca_base_framework_t opal_smsc_base_framework;
-extern mca_smsc_component_t *selected_component;
 extern mca_smsc_module_t *selected_module;
```

Comment:
What about the selected_module symbol? Should it be renamed to `opal_smsc_base_selected_module`?

Reply:
selected_module is not used anywhere, so is deleted.